packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -60,3 +67,107 @@ abstract class BuildSubCommand extends FlutterCommand {
     }
   }
 }
+
+Future<Null> updateVersion({@required BuildInfo buildInfo}) {
```

Comment:
`Future<void>`

Reply:
@tvolkert what is the different between `Future<Null>` and `Future<void>`?

Reply:
found it https://stackoverflow.com/questions/49410139/dart-2-difference-between-futurevoid-and-futurenull